app/src/main/java/com/pvp/app/ui/common/Buttons.kt

```diff
@@ -60,7 +61,7 @@ fun Button(
 @Composable
 fun ButtonConfirm(
     border: BorderStroke? = null,
-    confirmationButtonContent: @Composable RowScope.() -> Unit = { Text("Proceed") },
+    confirmationButtonContent: @Composable RowScope.() -> Unit = { Text("Proceed", fontWeight = FontWeight.Bold) },
```

Comment:
I believe lambda body should be split into multiple lines, because Text has multiple parameters. There is `projectStyles.xml` under `./app`, try to import it to your IDE's Code Style Kotlin settings, `CTRL+ALT+L` would help you here.